adapters/rubicon/rubicon_test.go

```diff
@@ -1161,6 +1133,90 @@ func TestOpenRTBRequestWithImpAndAdSlotIncluded(t *testing.T) {
 		"Unexpected dfp_ad_unit_code: %s", rubiconExtInventory["dfp_ad_unit_code"])
 }
 
+func TestOpenRTBRequestWithSpecificExtUserEids(t *testing.T) {
+	SIZE_ID := getTestSizes()
+	bidder := new(RubiconAdapter)
+
+	request := &openrtb.BidRequest{
+		ID: "test-request-id",
+		Imp: []openrtb.Imp{{
+			ID: "test-imp-id",
+			Banner: &openrtb.Banner{
+				Format: []openrtb.Format{
+					SIZE_ID[15],
+					SIZE_ID[10],
+				},
+			},
+			Ext: json.RawMessage(`{"bidder": {
+				"zoneId": 8394,
+				"siteId": 283282,
+				"accountId": 7891
+			}}`),
+		}},
+		User: &openrtb.User{
+			Ext: json.RawMessage(`{"eids": [
+                {
+                    "source": "pubcid",
+                    "id": "2402fc76-7b39-4f0e-bfc2-060ef7693648"
+				},
+				{
+                    "source": "adserver.org",
+                    "uids": [{
+                        "id": "3d50a262-bd8e-4be3-90b8-246291523907",
+                        "ext": {
+                            "rtiPartner": "TDID"
+                        }
+                    }]
+                },
+				{
```

Comment:
Just a nit pick but can you please just fix the indentation for this json blob?

Reply:
Sure, thanks.